ui-admin/src/portal/siteContent/SiteContentLoader.tsx

```diff
@@ -47,7 +47,7 @@ const SiteContentLoader = ({ portalEnvContext }: {portalEnvContext: PortalEnvCon
       Store.addNotification(failureNotification('save failed'))
       return
     }
-    await switchToVersion(newVersion.id, newVersion.stableId, newVersion.version, newVersion)
+    await switchToVersion(newVersion.id, newVersion.stableId, newVersion.version, undefined)
```

Comment:
This fixes a previous bug that caused all pages other than the landing page to be blank after saving a new version. `switchToVersion` allows you to pass in previously-loaded content or force it to fetch the version from the server. Here we were using the response from `createNewSiteContentVersion` to avoid going back to the server to fetch the newly-saved version. Sadly that response only includes the htmlPage contents for the landing page, hence the blank pages for everything else. It's cheap enough to just fetch the version after saving it but I could have also updated the response for that endpoint.

Reply:
Bug fixed _for real_ courtesy of Devon 35cbfa074001955a980e9145b94bb0122b9fa43f